include/deal.II/lac/sparsity_pattern.h

```diff
@@ -928,6 +928,15 @@ class SparsityPattern : public Subscriptor
   void print_gnuplot (std::ostream &out) const;
 
   /**
+   * Prints the sparsity of the matrix in a .svg file which can be opened in a web browser.
+   * The .svg file contains pixels which correspond to the entries in the matrix. An entry
+   * in the matrix which contains a non-zero value corresponds with a red pixel while a
+   * zero-valued entry in the matrix correspond with a white pixel.
```

Comment:
A pixel is really a physical representation of an element of your screen, or alternatively the smallest, indivisible piece of a picture. Pixels can't be scaled --  I think you want to call them "squares" in the documentation.